### PR TITLE
Vespa client feed and query

### PR DESCRIPTION
**Pull request by farshidz**
Vespa client feed and query functionality implementation

### CODE REVIEW

File: src/marqo/vespa/vespa_client.py [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+            raise VespaError(e) from e
+
+        self._raise_for_status(resp)
+
+        return QueryResult(**resp.json())
+
+    def feed_batch(self,
+                   batch: List[VespaDocument],
+                   schema: str,
+                   concurrency: int = 100,
```
Comment (pandu-k) [inline src/marqo/vespa/vespa_client.py]: is there a risk of adding excess network pressure? especially if end users make many concurrent add_docs calls

Comment (farshidz) [reply src/marqo/vespa/vespa_client.py]: There is and at some point Vespa starts rejecting connections. Though less likely once you add inference to the mix. I've reduced the default to 10. We'll have to fine-tune this based on the Vespa cluster and also Marqo inference cluster size.

File: scripts/vespa_local/schemas/marqo_settings.sd
Comment (pandu-k) [inline scripts/vespa_local/schemas/marqo_settings.sd:1]: What is the purpose of the vespa_local dir? Is this the copy of the remote vespa app package? or is this for local-README style deployments?

**Review by pandu-k — APPROVED** [review]

File: scripts/vespa_local/schemas/marqo_settings.sd
Comment (farshidz) [reply scripts/vespa_local/schemas/marqo_settings.sd:1]: Currently this is to help with local Vespa so the latter. It's unlikely we'll keep these in the final state but right now they help with testing and development

**Pull request merged**